lib/args.js

```diff
@@ -24,13 +25,17 @@ var nopt = require('nopt'),
         "json" : "Path to be scanned",
         "version" : "Show the current version",
         "export" : "Export format, 'tree' or 'json'",
-        "help" : "This help"
+        "help" : "This help",
+        "include": "Set this to dependencies or devDependencies",
```

Comment:
There are [peer dependencies](https://docs.npmjs.com/files/package.json#peerdependencies), bundled dependecies and optional dependencies too.